README.md

```diff
@@ -4,14 +4,23 @@ This repo implements a shared library for two javascript environments
 to create an encrypted and authenticated communication channel, by
 sharing a secret key and by relaying messages through a websocket server.
 
-It will be used by the Firefox Accounts pairing flow, with one side
-of the channel being web content from https://accounts.firefox.com and
+It is used by the [Firefox Accounts pairing flow](
+https://mozilla.github.io/ecosystem-platform/docs/features/firefox-accounts/pairing),
+with one side of the channel being web content from https://accounts.firefox.com and
 the other side of the channel being a signed-in Firefox instance.
 
 
 API
 ===
 
+To use this library, each side needs to agree on the URL of a
+[channelserver](https://github.com/mozilla-services/channelserver)
+instance which they can use to exchange messages via WebSocket:
+
+```
+const CHANNEL_SERVER_URL = "wss://dev.channelserver.nonprod.cloudops.mozgcp.net"
```

Comment:
I haven't checked this in a while but I remember this not working and having to switch to https://github.com/mozilla/fxa/blob/e1fcec815dc52d81f1633528c43d434cd8dab058/packages/fxa-content-server/server/lib/configuration.js#L451 in content-server.